CLAUDE.md

```
@@ -0,0 +1,62 @@
+- name: camelCase
+
+#### Event Handler
+
+- handle + 대상 + 동작
+- 예: handleLoginSubmit, handleInputChange
+
+### 3. API Naming
+
+- prefix: get / create / update / delete
+
+  ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)